graf3d/eve/inc/TEveVector.h

```diff
@@ -24,15 +26,23 @@ class TVector3;
 template <typename TT>
 class TEveVectorT
 {
+   /// The std::array guarantees that the data is contiguous in memory and we can implement safely Arr().
+   std::array<TT, 3> fXYZ {{0}};
 public:
-   TT fX, fY, fZ; // Components of the vector.
+   TT &fX = *&fXYZ[0], fY = *&fXYZ[1], fZ = *&fXYZ[3]; // Components of the vector.
```

Comment:
I can't reply to Axels' comment below for some reason ... this would indeed be the best option, at least until somebody invents 48-bit architectures :)